android/src/com/frostwire/android/gui/activities/SettingsActivity.java

```diff
@@ -98,12 +99,21 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         addPreferencesFromResource(R.xml.application_preferences);
 
-        getListView().setPadding(20, 0, 20, 0);
+        LinearLayout root = (LinearLayout) findViewById(android.R.id.list).getParent().getParent().getParent();
```

Comment:
hehehe, what is this hack about?...let's find the **right** way to do this, calling `getParent()` three times is very weak at best.